integrations/flink_connector/sample-kinesis-to-timestream-app/src/main/java/com/amazonaws/samples/kinesis2timestream/TimestreamInitializer.java

```diff
@@ -15,31 +15,23 @@
  * Checks if required database and table exists in Timestream. If they do not exists, it creates them
  */
 public class TimestreamInitializer {
-    private final TimestreamWriteClient writeClient;
```

Comment:
Just wondering for user case the TimestreamInitializer object is not destroyed? If not destroyed, the writeClient is not closed. If it is destroyed, writeClient should be closed in the destruction phase.

Reply:
@RoyZhang2022 , in java we don't need to destroy objects. In the example we use try(TimestreamWriteClient client = createClient()) that equal to 
TimestreamWriteClient client = createClient()
try() {
//some logic
} finally {
client.close();
}